WLC/benchmark.py

```diff
@@ -1,10 +1,16 @@
+import logging
 import re
 
+from WLC.__main__ import FORMAT
```

Comment:
Do you think importing `WLC.__main__` is pythonic? Import this from a util file maybe?

Reply:
Moved to utils.